enhancements/suspend-and-resume-scan-schedule.md

```diff
@@ -0,0 +1,231 @@
+the `ScanSettingBinding` entirely.
+
+This enhancement describes the need and implementation details for a more
+ergonomic way of suspending and resuming scan schedules.
+
+## Motivation
+
+While making a cluster compliant, there may be times when cluster administrator
+do not want to run a particular scan, but they don't necessarily want to
+completely delete the `ScanSettingBinding` either.
+
+For example, imagine there is a planned maintenance window set for when a
+compliance scan is scheduled to run. If the scan runs, it will return
+inaccurate results. Another example is if an organization is working through
+compliance findings and wants to prevent subsequent scans until particular
+actions are taken.
+
+To deal with this, cluster administrators can either delete the binding, or
+manually pause the `CronJob` using the `Suspend` attribute.
+
+Deleting the binding is arguable heavy-handed, and requires users to put it
+back when they're ready to resume scanning. Conversely, digging into the
+rerunner implementation details of `CronJob` isn't ideal because it requires
+underlying knowledge about that particular Kubernetes feature.
+
+The Compliance Operator can help here by orchestrating a change to pause
+`CronJob` rerunner for applicable scans, and expose those details to cluster
+administrators.
+
+### Goals
+
+The goal of this work is to make it easy for a cluster administrator to suspend and resume
+scan schedules with a single API call. For example, suspendingan existing scan
+should be as modifying a detail of the scan. Similarly, resuming a scan
+should be as easy updating a scan attribute.
+
+### Non-Goals
+
+This enhancement is not focused on stopping a scan that's already in flight. It
+is only focused on suspending a scan that is going to take place in the future.
+
+## Proposal
+
+### User Stories
+
+- As a cluster administrator, I can suspend a `ComplianceScan`, preventing it from automatically running according to its predefined schedule.
+- As a cluster administrator, I can resume a paused `ComplianceScan`, allowing it to run automatically according to its predefined schedule.
+
+By default, all scans are in an active state, meaning they are schedule to run unless explicitly suspended.
+
+### API Extensions
+
+This change will be implemented similar to how rescans are implemented, which rely on annotations.
+
+Cluster administrators initiate an on-demand scan using an annotation:
+
+```console
+$ oc -n openshift-compliance annotate compliancescans/cis-ocp4-worker compliance.openshift.io/rescan=
+```
+
+We can use a similar approach for suspending a scan schedule:
+
+```console
+$ oc -n openshift-compliance annotate compliancescans/cis-ocp4-worker compliance.openshift.io/suspend=
+```
+
+Additionally, cluster administrators can remove the annotation to resume the scan:
+
+```console
+$ oc -n openshift-compliance annotate compliancescans/cis-ocp4-worker compliance.openshift.io/suspend-
```

Comment:
I wonder if suspend should be applied to SSB instead and propagated to the scan/suite object? When we have updates to the cluster, if some suite/scan objects are removed during the update, a new scan will be launched regardless if this annotation is only applied on the scan/suit object.

Reply:
Ok - I see what you mean. I'll update to use the `ScanSettingBinding` instead.